src/Controller/Api/CategoryController.php

```diff
@@ -89,7 +91,7 @@ public function __construct(CategoryManagerInterface $categoryManager, FormFacto
      *     @SWG\Response(
      *         response="200",
      *         description="Returned when successful",
-     *         @SWG\Schema(ref=@Model(type="Sonata\DatagridBundle\Pager\PagerInterface"))
+     *         @SWG\Schema(ref=@Model(type=BasePager::class, groups={"sonata_api_read"}))
```

Comment:
Never seen groups for the operation (read / write) in swagger.

Would prefer groups for the different entities (category, collection, tag, ...)